spec/features/db/draft_works_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "spec_helper"
+
+describe "Annict DB" do
+  describe "作品の編集リクエスト作成ページ" do
+    let(:user) { create(:registered_user) }
+
+    before do
+      login_as(user, scope: :user)
+      visit "/db/draft_works/new"
+    end
+
+    it "ページが表示されること" do
+      expect(page).to have_content("作品情報")
+    end
+
+    context "入力して送信したとき" do
+      before do
+        within("#new_draft_work") do
+          fill_in "draft_work[title]", with: "のんのんびより"
+          select "TV", from: "draft_work[media]"
+          fill_in "draft_work[edit_request_attributes][title]", with: '「のんのんびよりを登録」'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.